examples/src/index.js

```diff
@@ -516,10 +520,18 @@ const StyledDemoPage = styled(DemoPage)`
     color: rgba(255, 255, 255, 0.9);
     font-size: 34px;
     margin-top: -54px;
+    margin-bottom: 10px;
```

Comment:
don't like those magic numbers too much, but I guess not much to do about them

Reply:
Yeah - it's a little thrown-together. Personally I'd like to redesign the layout a bit, put in a side bar and a cleaner title.